mne/preprocessing/eyetracking/eyetracking.py

```diff
@@ -165,3 +170,128 @@ def _convert_mm_to_m(array):
 
 def _convert_deg_to_rad(array):
     return array * np.pi / 180.0
+
+
+def convert_units(inst, calibration, to="radians"):
+    """Convert Eyegaze data from pixels to radians or vice versa.
+
+    Parameters
+    ----------
+    inst : instance of Raw, Epochs, or Evoked
+        The Raw, Epochs, or Evoked instance with eyegaze channels.
+    calibration : mne.preprocessing.eyetracking.Calibration
+       instance of  :class:`~mne.preprocessing.eyetracking.Calibration`, containing
+        information about the screen size and viewing distance (in meters), and
+        the screen resolution (in pixels).
```

Comment:
A shorter type with a xref alias would be nicer:

```
calibration : Calibration
   (...)

or

calibration : Eye-tracking Calibration
    (...)
```

with an alias for the chosen name in the documentation configuration: https://github.com/mne-tools/mne-python/blob/5df4cd6506ca2fb244070865a92bdbba8dabc1c4/doc/conf.py#L198-L275

Reply:
Thx @mscheltienne , aliases were a bit confusing for me to figure out. I should have some time to return to this next week.